hyperspace/Makefile

```diff
@@ -0,0 +1,38 @@
+module-hyperspace=hyperspace
+composable_centauri_image="composablefi/composable-centauri:latest"
+hyperspace_image="composablefi/hyperspace:latest"
+
+.PHONY: run-setup-hyperspace stop-setup-hyperspace build-release-hyperspace tests-hyperspace
+.PHONY: build-docker-image-hyperspace publish-docker-image-hyperspace
+
+run-setup-hyperspace:
+	docker run --platform linux/amd64 \
+	-d \
+	--rm \
+	-ti \
+	--name composable_centauri \
+	-u1000:1000 \
+	-p9944:9944 \
+	-p9188:9188 \
+	-p9988:9988 \
+	$(composable_centauri_image)
```

Comment:
You probably want to run the `scripts/parachain-launch/docker-compose.yml` not just a single node running which won't produce any blocks